buildpipeline/Dotnet-CoreClr-Trusted-BuildTests.json

```diff
@@ -279,6 +408,9 @@
     "PB_ContainerName": {
       "value": "$(Label)-$(TestContainerSuffix)-test"
     },
+    "PB_FeedReadAccessToken": {
+      "value": "$(PB_FeedReadAccessToken)"
```

Comment:
You can just leave this blank here.  VSTS might even complain about circular reference the way that you have it.

Reply:
Since I'll remove the creation of an auth'd nuget.config this goes away with that